src/backend/api/meals.js

```diff
@@ -1,15 +1,97 @@
 const express = require("express");
 const router = express.Router();
 const knex = require("../database");
-
-router.get("/", async (request, response) => {
-  try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
-  } catch (error) {
-    throw error;
-  }
+// Returns all meals
+router.get("/", async(request, response) => {
+    try {
+        // knex syntax for selecting things. Look up the documentation for knex for further info
+        const titles = await knex("meals");
+        response.json(titles);
+    } catch (error) {
+        throw error;
+    }
```

Comment:
It doesn't really work if you have defined the same route -> .get("/") twice!
Nodejs doesn't know which function should be executed, even if in one case you're sending in query parameters and in the other not.
Try to merge them!